pyvista/plotting/charts.py

```diff
@@ -150,6 +201,20 @@ def width(self):
     def width(self, val):
         self.SetWidth(val)
 
+    @property
+    def pen(self):
+        """
+        :return: Retrieve vtkPen object controlling how lines in this plot are drawn.
+        """
```

Comment:
We're using the numpydoc documentation style

```suggestion
        """vtkPen object controlling how lines in this plot are drawn.

        Returns
        --------
        vtk.vtkPen
            `vtkPen` object controlling how lines in this plot are drawn.

        Examples
        ---------
        >>> import pyvista
        """
```
All public methods should have documented methods, and if possible, include plots.  These are now automatically rendered (you may have to merge with main).  These are great for demonstrating functionally.  For example:
https://dev.pyvista.org/api/core/_autosummary/_autosummary/pyvista.PolyDataFilters.boolean_difference.html

Reply:
Ok, that's really cool, will do that! I will focus more on the documentation in future commits, once the main implementation is done and the interface is +- fixed